tools/ci_build/github/windows/helpers.ps1

```diff
@@ -392,6 +392,10 @@ function Install-Protobuf {
         exit $exitCode
     }
     $cmake_args = "--build", ".",  "--parallel", "--config", $build_config, "--target", "INSTALL"
+    if ($use_cache)
+    {
```

Comment:
nit: bracing is inconsistent (opening brace is on same line as `if` elsewhere)

Reply:
thx. fixed